src/DataCollector/GateCollector.php

```diff
@@ -17,14 +17,9 @@
  */
 class GateCollector extends MessagesCollector
 {
-    /** @var int */
-    protected $backtraceLimit = 15;
```

Comment:
After this change, link editor always shows GateCollector.php
Before
<img width="262" height="78" alt="image" src="https://github.com/user-attachments/assets/26a16131-07d7-43d8-829e-b7b890b9ee95" />
After
<img width="256" height="66" alt="image" src="https://github.com/user-attachments/assets/0b6be039-1972-471c-bfd7-bae6c7f1fc19" />